tests/telemetry/conftest.py

```diff
@@ -36,11 +35,46 @@ def verify_telemetry_dockerimage(duthosts, enum_rand_one_per_hwsku_hostname):
     """
     docker_out_list = []
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
-    docker_out = duthost.shell('docker images docker-sonic-telemetry', module_ignore_errors=False)['stdout_lines']
+    docker_out = duthost.shell('docker images docker-sonic-gnmi', module_ignore_errors=False)['stdout_lines']
     docker_out_list = get_list_stdout(docker_out)
-    matching = [s for s in docker_out_list if b"docker-sonic-telemetry" in s]
+    matching = [s for s in docker_out_list if b"docker-sonic-gnmi" in s]
     if not (len(matching) > 0):
-        pytest.skip("docker-sonic-telemetry is not part of the image")
+        pytest.skip("docker-sonic-gnmi is not part of the image")
```

Comment:
Shouldn't we check if either exists, maybe docker-sonic-telemetry is running and docker-sonic-gnmi is not?

Reply:
telemetry docker image is build from gnmi docker image, so it's not possible.